content/chapter-03/ApplicationMetrics.md

```diff
@@ -0,0 +1,21 @@
+# Application Metrics
+
+## What to measure on an application?
+
+* Application level metrics ([Resource](https://stackify.com/application-performance-metrics/)):
+  * Error rate, e.g.: Internal error logged per second, 5XX requests/second
+  * Request rate, e.g.: Total requests per second, 2XX requests/second
+  * Response time, e.g.: Average response time, peak response time per second
+  * Availability percentage, e.g.: Percentage of uptime for interval of time
+  * Concurrent users, e.g.: Concurrent user count distribution over a day
+  * User satisfaction using [Apdex score](https://www.apdex.org/)
+  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per using of time.
+  
+  
+* System level metrics ([Resource]())
+  * CPU utilization, e.g.: CPU utilization percentage per second
```

Comment:
Is the CPU measured per sec?

Reply:
The idea I want to put is that the measurement is always per time unit unless you just sampling it. It is not common though. Usually is sampled so I will remove that.